examples/src/multi-file/endpoints/find-users.ts

```diff
@@ -0,0 +1,19 @@
+import { api, endpoint, Optional, queryParam } from "@zenclabs/spot";
+
+@api()
+export class Api {
+  @endpoint({
+    method: "GET",
+    path: "/users"
+  })
+  getUsers(
+    @queryParam limit: number
```

Comment:
Is there a specific reason for making this different from `single-file`? (it's not optional here)

To clarify: the two folders were meant to be equivalent.

Reply:
there is no good reason, just want to have example, I will update it